client/layout/community-translator/launcher.jsx

```diff
@@ -4,8 +4,7 @@
  * External dependencies
  */
 
-import PropTypes from 'prop-types';
-import { localize } from 'i18n-calypso';
+import i18n, { localize } from 'i18n-calypso';
```

Comment:
The removal of `PropTypes` here happens to show me one more flaw not specifically related to this PR. `<TranslatorLauncher />` component actually has the prop, `translate()`, that we should have defined explicitly. Even though it comes from the `localize()` call, but the component itself can't really tell the difference of getting it from prop passing or being wrapped by a higher-order component. A good example is [how ellipsis menu does it](https://github.com/Automattic/wp-calypso/blob/master/client/components/ellipsis-menu/index.jsx#L22).

It's not related to this PR, but just wanted to mention it :)

Reply:
Thank you for telling me this. Didn't notice that there is this prop for`<TranslatorLauncher />`! Added it back to the PropTypes list in d63d3bcb1c628ffdabfbc4e7a3f579721612f1ca.